root_github.tf

```diff
@@ -631,3 +631,46 @@ module "github_file_upload_data_repository" {
     WORKFLOW_PAT       = module.common_ssm_parameters.params[local.github_access_token_name].value
   }
 }
+
+module "github_backend_checks_utils_environment" {
```

Comment:
Should this resource be called `github_backend_checks_utils_repository`  to keep it consistent with the other resource names?

Reply:
Yep, must have copied over from terraform-environments :smile: I've changed it.